examples/parallel/cpp/smartredis_mnist.cpp

```diff
@@ -104,8 +104,7 @@ int main(int argc, char* argv[]) {
     logger_name += std::to_string(rank);
 
     // Initialize a Client object
-    bool cluster_mode = true; // Set to false if not using a clustered database
-    SmartRedis::Client client(cluster_mode, logger_name);
+    SmartRedis::Client client(logger_name);
```

Comment:
Especially if we want to encourage people to use the new route using ConfigOptions, we should update these examples. I'd actually be ok doing this in a separate PR as I think it might be a good educational task to do this and write a 'migration guide' for users since this is a fairly fundamental API break.

Reply:
I agree both that this should be done and that it's a separate PR